source/includes/scripting/fact-require-ems-modules.rst

```diff
@@ -0,0 +1,24 @@
+There are two packaging standards for Node.js modules.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Packaging Standard
+     - Works with require()
+
+   * - ``CommonJS`` (CJS)
+     -  Yes
+
+   * - ``ECMAScript``
+     -  No
+
+You cannot ``require()`` an ECMAScript module in ``mongosh``. If you
+want to use functionality from an ECMAScript module, check to see if
+there is a CommonJS version that you can use instead. For more
+information, see:
+
+- `Node.js module documentation
+  <https://nodejs.org/api/esm.html#modules-ecmascript-modules>`__
+- `npm package installation documentation
+  <https://docs.npmjs.com/cli/v6/commands/npm-install>`__
```

Comment:
I’m not quite sure how the npm documentation relates to this topic – installation and usage of modules are fairly separated things these days. If you do want to link the npm docs here, I’d pick a more recent version, since npm@6.x is fairly outdated at this point.